CppCoreGuidelines.md

```diff
@@ -17997,23 +17997,23 @@ This leads to longer programs and more errors caused by uninitialized and wrongl
 
 ##### Example, bad
 
-   int use(int x)
-   {
-       int i;
-       char c;
-       double d;
-
-       // ... some stuff ...
-
-       if (x < i) {
-           // ...
-           i = f(x, d);
-       }
-       if (i < x) {
-           // ...
-           i = g(x, c);
-       }
-       return i;
+    int use(int x)
+    {
+        int i;
+        char c;
+        double d;
+ 
+        // ... some stuff ...
+ 
```

Comment:
seems even more have failed. I'll revert my 2 commits, and try by just changing the first 2 lines of the example, and see what CI does with those